Buttplug.Client/ButtplugWSClient.cs

```diff
@@ -110,19 +107,23 @@ public ButtplugWSClient(string aClientName)
                 throw new AccessViolationException("Already connected!");
             }
 
-            _ws = new ClientWebSocket();
+            _ws = new WebSocket(aURL.ToString());
```

Comment:
As of this change, this function is no longer async, but is still marked as such. I think we could make it awaitable from the outside by wrapping the evented portions of WebSocket4Net though. See comments below.